client/src/modules/debtors/groups.update.html

```diff
@@ -23,9 +23,19 @@
 
           <div
             class="form-group has-feedback"
-            ng-class="{'has-error' : debtorGroup.color.$invalid && debtorGroup.$submitted}">
+            ng-class="{'has-error' : !GroupUpdateCtrl.group.color && debtorGroup.$submitted}">
             <label class="control-label" translate>FORM.LABELS.COLOR</label>
-            <input name="color" id="title-color" type="color" ng-model="GroupUpdateCtrl.group.color" class="form-control" ng-value="GroupUpdateCtrl.group.color">
+
+            <ui-select tagging ng-model="GroupUpdateCtrl.group.color">
+              <ui-select-match placeholder="Pick one...">
+                <i class="fa fa-circle" style="color: {{$select.selected.value}}; font-sze:14px"></i>
+                &nbsp;<span translate>{{$select.selected.name}}</span>
+              </ui-select-match>
+              <ui-select-choices repeat="val.value as val in GroupUpdateCtrl.colors | filter: {'name' :$select.search}">
+                <div>
+                  <span><i class="fa fa-circle" style="color: {{val.value}}; font-sze:14px"></i>&nbsp; <span translate>{{val.name}} </span> </span>
```

Comment:
In the future, it would be better to use [a ngStyle tag](https://docs.angularjs.org/api/ng/directive/ngStyle) if you will interpolate on a style element.  Something like this:
```html
<i class="fa fa-circle" ng-style="{ color : val.value }">
```